File: app/src/main/java/com/osrsd/cache/def/HealthbarDefinition.java
package com.osrsd.cache.def;

import lombok.Data;

@Data
public final class HealthbarDefinition implements Definition {

    private int id;
    private int field3276;
    private int field3277;
    private int field3278;
    private int field3283;
    private int field3272;
    private int field3275;
    private int healthBarFrontSpriteId;
    private int healthBarBackSpriteId;
    private int healthScale;
    private int healthBarPadding;

    public HealthbarDefinition(int id) {
        this.id = id;
        field3277 = 255;
        field3278 = 255;
        field3283 = -1;
        field3272 = 1;
        field3275 = 70;
        healthBarFrontSpriteId = -1;
        healthBarBackSpriteId = -1;
        healthScale = 30;
        healthBarPadding = 0;
    }

}

File: app/src/main/java/com/osrsd/spring/domain/HealthBarDefinition.kt
package com.osrsd.spring.domain

import javax.persistence.Entity
import javax.persistence.Id

@Entity
data class HealthBarDefinition(@Id private val id: Int? = 0) : Definition {
    var field3276: Int = 0
    var field3277: Int = 255
    var field3278: Int = 255
    var field3283: Int = -1
    var field3272: Int = 1
    var field3275: Int = 70
    var healthBarFrontSpriteId: Int = -1
    var healthBarBackSpriteId: Int = -1
    var healthScale: Int = 30
    var healthBarPadding: Int = 0
}
